doc/source/whatsnew/v0.25.0.rst

```diff
@@ -258,7 +258,10 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+DataFrame
+^^^^^^^^^^^
 
+- Fixed a bug where Constructing a :class:`DataFrame` using a ``range`` (e.g. ``pd.DataFrame(range(3))``) would cause an ``AttributeException`` (:issue:`26342`).
```

Comment:
Bug in construction of a ``:class:`DataFrame` `` using a ``range`` object (:issue:....)

no need to be more verbose; we rarely mention the original error received